content/en/docs/tasks/security/authentication/mtls-migration/index.md

```diff
@@ -30,16 +30,81 @@ down once the migration is done.
 
 * Understand Istio [authentication policy](/docs/concepts/security/#authentication-policies) and related [mutual TLS authentication](/docs/concepts/security/#mutual-tls-authentication) concepts.
 
+* Read the [authentication policy task](/docs/tasks/security/authentication/authn-policy) to
+  learn how to configure authentication policy.
+
 * Have a Kubernetes cluster with Istio installed, without global mutual TLS enabled (e.g use the demo configuration profile as described in
 [installation steps](/docs/setup/getting-started), or set the `global.mtls.enabled` installation option to false).
 
-* For demo
-    * Create the following namespaces and deploy [httpbin]({{< github_tree >}}/samples/httpbin) and [sleep]({{< github_tree >}}/samples/sleep) with sidecar on both of them.
-        * `foo`
-        * `bar`
+* You need to make sure your cluster is in PERMISSIVE mode before migrating to mutual TLS.
+  Run the following command to check:
+
+    {{< text bash >}}
+    $ kubectl get meshpolicy default -o yaml
+    ...
+    spec:
+      peers:
+      - mtls:
+          mode: PERMISSIVE
+    {{< /text >}}
+
+  If you find the output same as above, you can skip the next step. Otherwise, move to the next step.
+
+* Run the following command to enable PERMISSIVE mode for the cluster. In general, this operation
+  does not cause any interruption your your workloads, but also see the warning message below.
+
+    {{< warning >}}
+    In PERMISSIVE mode, the Envoy sidecar relies on the
+    [ALPN](https://en.wikipedia.org/wiki/Application-Layer_Protocol_Negotiation) value _istio_ to
+    decide whether to terminate the mutual TLS traffic. If your workloads (without Envoy sidecar)
+    have enabled mutual TLS directly to the services with Envoy sidecars, enabling PERMISSIVE mode
+    may cause these connections to fail.
+    {{< /warning >}}
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: "authentication.istio.io/v1alpha1"
+    kind: "MeshPolicy"
+    metadata:
+      name: "default"
+    spec:
+      peers:
+      - mtls:
+          mode: PERMISSIVE
+    EOF
+    {{< /text >}}
+
+{{< tip >}}
+Use _istioctl authn tls-check_ to show the mTLS configuration of all connections from one pod:
+
+{{< text bash >}}
+$ istioctl authn tls-check \<YOUR\_POD\> -n \<YOUR\_NAMESPACE\>
+{{< /text >}}
+
+{{< /tip >}}
 
-    * Create the following namespace and deploy [sleep]({{< github_tree >}}/samples/sleep) without sidecar
-        * `legacy`
+In the following, the task is divided into two parts.
+
+* If you want to enable mutual TLS for your workloads one after one, you can go to
+[this section](/docs/tasks/security/authentication/mtls-migration/#enable-mutual-tls-for-a-service),
```

Comment:
nit: it's easier to quickly understand the option if you put the actual title of the section instead of using `this section` here.

Reply:
Sure, done.